pkg/secrets/secrets.go

```diff
@@ -271,12 +272,19 @@ func validateSecretName(name string) error {
 
 // getDriver creates a new driver.
 func getDriver(name string, opts map[string]string) (SecretsDriver, error) {
-	if name == "file" {
+	switch name {
+	case "file":
 		if path, ok := opts["path"]; ok {
 			return filedriver.NewDriver(path)
 		} else {
 			return nil, errors.Wrap(errInvalidDriverOpt, "need path for filedriver")
 		}
+	case "pass":
+		if bin, ok := opts["binary"]; ok {
+			return passdriver.NewDriver(bin)
+		} else {
+			return passdriver.NewDriver("gopass")
```

Comment:
I have a slight preference for using `pass` as the default.  It has a higher availability.